### PR TITLE
feat: add demo talkingdata

### PR DESCRIPTION
**Pull request by vagetablechicken**
**before 050, docker test will fail.**

### CODE REVIEW

File: .github/workflows/openmldb-docker.yml [hunk 1]
```diff
@@ -6,13 +6,13 @@ on:
       - main
     paths:
       - .github/workflows/openmldb-docker.yml
-      - demo/predict-taxi-trip-duration-nb/**/*
+      - demo/*
```
Comment (aceforeverd) [inline .github/workflows/openmldb-docker.yml]: demo/**/*

**Review by aceforeverd — APPROVED** [review]

File: demo/predict-taxi-trip-duration/README.md [hunk 2]
```diff
@@ -28,11 +28,12 @@ w2 as (PARTITION BY passenger_count ORDER BY pickup_datetime ROWS_RANGE BETWEEN
 
 **Start docker**
 ```
-docker run -it 4pdosc/openmldb:0.4.1 bash
+docker run -it 4pdosc/openmldb:0.4.4 bash
 ```
 **Initialize environment**
 ```bash
 ./init.sh
+cd taxi-trip
```
Comment (dl239) [inline demo/predict-taxi-trip-duration/README.md:36]: - update the standalone steps and [quickstart docs](https://github.com/4paradigm/OpenMLDB/blob/main/docs/zh/quickstart/openmldb_quickstart.md)

File: demo/talkingdata-adtracking-fraud-detection/README.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+cd /work/talkingdata
+```
+
+#### Run locally
+
+Download OpenMLDB server pkg, version >= 0.5.0 .
+
+Install all dependencies:
+```
+pip install pandas xgboost==1.4.2 tornado "openmldb>=0.5.0"
```
Comment (dl239) [inline demo/talkingdata-adtracking-fraud-detection/README.md:23]: these libs have been installed when docker building. it is need to install again?

File: demo/talkingdata-adtracking-fraud-detection/README.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+## Process
+
+### Start OpenMLDB cluster
+```
+./init.sh
+```
+
+### Train and Serve
+```
```
Comment (dl239) [inline demo/talkingdata-adtracking-fraud-detection/README.md:44]: `cd talkingdata-adtracking-fraud-detection`?

File: demo/talkingdata-adtracking-fraud-detection/README.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+python3 train_and_serve.py
+```
+We use OpenMLDB to do feature extraction, and train by xgboost, see `train_and_serve.py`.
```
Comment (dl239) [inline demo/talkingdata-adtracking-fraud-detection/README.md]: add link to `train_and_serve.py`

File: demo/talkingdata-adtracking-fraud-detection/README.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+```
+2. predict once
+```
+python3 predict.py
+```
+
+You can run `pkill python3` to kill the predict server in the background.
+
+## Q&A
+Q: train_and_serve.py core dump at SetGPUAttribute...
```
Comment (dl239) [inline demo/talkingdata-adtracking-fraud-detection/README.md:73]: add blank line

File: demo/talkingdata-adtracking-fraud-detection/README.md (hunk 3 above)
Comment (vagetablechicken) [reply demo/talkingdata-adtracking-fraud-detection/README.md:23]: It's for ' Run locally'

**Review by dl239 — APPROVED** [review]

**Pull request merged**